src/Parser/Hunter/BeastMastery/Modules/Spells/BestialWrath/BestialWrath.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS/index';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import { formatNumber } from 'common/format';
+import getDamageBonus from "Parser/Hunter/Shared/Core/getDamageBonus";
+import StatisticBox from "Main/StatisticBox";
+import SpellIcon from "common/SpellIcon";
+import ITEMS from "common/ITEMS/HUNTER";
+
+const T204P_MODIFIER = 0.15;
+const T20_2P_MODIFIER_PR_STACK = 0.015;
+const BESTIAL_FURY_MODIFIER = 0.15;
+const BESTIAL_WRATH_BASE_MODIFIER = 0.25;
+let BESTIAL_WRATH_TOTAL_MODIFIER = 0;
```

Comment:
Shouldn't this be a class property since it's used by an instance of BestialWrath? Running multiple instances would run into problems.

Reply:
I'm not sure as to what you mean, but I made a small change in how I utilise this in the latest push

Reply:
you should `let bestialWrathBonus` inside the class so it's an instance variable, and you should set its value exactly once in an `on_initialized()` function. Then probably also have a `currentBestialWrathBonus` value that's set to bestialWrathBonus on cast, and is then boosted throughout.